server/src/main/java/org/elasticsearch/action/support/RetryableAction.java

```diff
@@ -194,7 +192,7 @@ public void onFailure(Exception e) {
                     long millisExceedingTimeout = delayMillis + elapsedMillis - timeoutMillis;
```

Comment:
Can we change this to: `delayMillis - remainingMillis`? That makes the `delayMillis` assignment a few lines down more obvious.